src/components/notification/notification.vue

```diff
@@ -1,18 +1,35 @@
 <template>
-  <component :is="clickable ? 'button' : 'div'" :class="`apos-notification ${modifier}`" role="alert">
+  <component 
+    :is="clickable ? 'button' : 'div'" 
+    :class="modifierClass" 
+    :role="clickable ? false : 'alert'"
+    class="apos-notification"
```

Comment:
This pattern of the class binding being separate might risk missing one or the other. I'd suggest something like
```
  computed: {
    classList () {
      const classes = ['apos-notification'];

      const modifiers = [];
      if (this.type) {
        classes.push(`apos-notification--${this.type}`);
      }
      return classes;
    },
```

Then we're only binding `:class="classList"`.  At the very least they should be next to one another to track.